modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -274,6 +275,30 @@ Example output plugin configuration:
     auto_create_group      On
 ```
 
+==== Syslog Configuration Parameters
+
+[%header%autowidth.spread,cols="a,a,a,a"]
+.Syslog Configuration Parameters
+|===
+| Key | Description | Required | Default Value
+| Host | Host name or IP address of the remote Syslog server | Yes |
+| Port | TCP or UDP port of the remote Syslog server | Yes | 514
+| Mode | The mode of the transport type | Yes | udp
+| Syslog format | Syslog protocol format | No | rfc5424
+| Syslog Maxsize | Integer value that sets the maximum number of bytes per message. | No | 1024 for rfc3164, 2048 for rfc5424
```

Comment:
the others don't have a period at the end, prob remove it

Reply:
Ok i will remove it